extension/src/NABfunctions.ts

```diff
@@ -1536,3 +1536,74 @@ export async function renumberALObjects(): Promise<void> {
     showErrorAndLog("Renumber AL objects", error as Error);
   }
 }
+export async function troubleshootFindTransUnitsWithoutSource(): Promise<void> {
```

Comment:
How about moving these functions to a separate file like `Troubleshooting.ts` or `TroubleshootingFunctions.ts`

Reply:
done